recipes/lielab/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import copy, get, rmdir
+import os
+
+required_conan_version = ">=1.52.0"
+
+class LielabConan(ConanFile):
+    name = "lielab"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sandialabs/Lielab"
+    description = "Lielab is a C++ library for numerical Lie-theory: Lie groups," \
+                  " Lie algebras, homogeneous manifolds, and various functions and algorithms" \
+                  " on these spaces."
+    topics = ("Lie-theory", "Lie-group", "Lie-algebra", "numerical", "header-only")
+    package_type = "header-library"
+    license = "MIT"
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    exports_sources = ("CMakeLists.txt",
+                       "LICENSE",
+                       "lielab/*")
+    
+    no_copy_source = True
+
+    def requirements(self):
+        self.requires("eigen/3.4.0")
+
```

Comment:
```suggestion

    def build_requirements(self):
        self.tool_requires("cmake/[>=3.23 <4]")
```
Need to specify CMake tool_requires when requiring CMake newer than v3.15.